lib/elixir/src/elixir_erl.erl

```diff
@@ -563,13 +563,22 @@ checker_chunk(#{definitions := Definitions, deprecated := Deprecated}, NoWarnUnd
       end
     end, [], Definitions),
 
+  AllExports = [
+    {{'__info__', 1}, #{kind => def, deprecated_reason => nil}},
+    {{module_info, 0}, #{kind => def, deprecated_reason => nil}},
+    {{module_info, 1}, #{kind => def, deprecated_reason => nil}}
```

Comment:
I wouldn't include those in the exports, honestly. They are always available, so that would increase the size for no reason. I would rather special case them in the checker (which is what we did for xref).

Reply:
Pushed with fix for this.